src/server/server-prod.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-console */
```

Comment:
probably we don't need two files for server
we can move this logic to the server-dev.js

and then use condition to choose needed part of code

```
 if(production) {
...
} else {
...dev code here
}
```

Reply:
done